configs/sot/siamese_rpn/siamese_rpn_r50_20e_otb100.py

```diff
@@ -26,52 +31,36 @@
     dict(type='SeqColorAug', prob=[1.0, 1.0]),
     dict(type='SeqBlurAug', prob=[0.0, 0.2]),
     dict(type='VideoCollect', keys=['img', 'gt_bboxes', 'is_positive_pairs']),
-    dict(type='ConcatVideoReferences'),
+    dict(type='ConcatSameTypeFrames'),
     dict(type='SeqDefaultFormatBundle', ref_prefix='search')
 ]
 # dataset settings
 data = dict(
     samples_per_gpu=16,
-    train=[
+    train=dict(dataset_cfgs=[
```

Comment:
The key  of `train` in `data` can be inherit from base config?

Reply:
Because the `train_pipeline` is different, the `dataset_cfgs` list can not inherit from the base conifg.